src/components/TextInput/TextInput.module.scss

```diff
@@ -51,7 +51,6 @@
 
     &:focus {
       outline: none;
-      background-color: var(--form-control-background-color);
```

Comment:
From what I could tell, this background wasn't necessary. Please let me know if this assumption is incorrect.